README.md

```diff
@@ -1,4 +1,4 @@
-**This repo will be archived during *October* as it is for practice only**
+**This repo will be archived during _October_ as it is for practice only**
```

Comment:
Why do we want this change? The result is the same.

Reply:
I'm just curious why do we want this change on `October`

Reply:
This might have been a `prettier` change. I've noticed in my own projects that `prettier` prefers underscores for italics.

Reply:
if that is true, wouldn't the `**` be `__`?

Reply:
Nope - `prettier` uses `**` for bold, still.

Reply:
I see. But I don't see the reason to change. Should we revert to `*`?

Reply:
Sure, though we'll potentially run in to this same issue if a future contributor runs Prettier on this file.

Reply:
Looking back Eddie would have merged something like this. I'll merge this PR